src/summon/browzine-summon-adapter.js

```diff
@@ -856,12 +838,21 @@ browzine.summon = (function() {
           }
         }
 
-        if (directToPDFUrl) {
-          var quickLinkElement = $(documentSummary).find(".docFooter .customPrimaryLinkContainer");
+        if ((directToPDFUrl || articleLinkUrl)) {
+          var intervals = 0;
 
-          if (quickLinkElement) {
-            quickLinkElement.remove();
-          }
+          (function poll() {
+            var quickLinkElement = $(documentSummary).find(".docFooter .availabilityFullText a[display-text='::i18n.translations.PDF']");
+
+            if (quickLinkElement.length > 0) {
+              quickLinkElement.remove();
+            } else {
+              if (intervals < 120) {
+                intervals++;
+                requestAnimationFrame(poll);
```

Comment:
One thought: instead of requestAnimationFrame, what if we did this with a setTimeout and only checked, say, at 100 msec increments for up to 20 seconds or something like that?

When I looked at other uses of requestAnimationFrame (like in Nomad), it seems like it goes really fast (I think firing around once per msec), although especially here I think we only need to check at a relatively infrequent rate (like once per 100 msec).

Thoughts?

Reply:
Intervals are capped, best to test in IT.